instructions/level1.md

```diff
@@ -252,11 +260,21 @@ To validate our form we will need to follow next steps:
   
   ```
   var errors = {
-    name: nameValidation(name)
+    name: nameValidationError(name)
   };
   ```
-  
+
   TODO: 4. Now do same for city and description.
+
+  When you are defining multiple keys for an object, you need to add a comma
+  to the previous value. For example:
```

Comment:
This might have been covered in the beginners tutorial (I haven't checked) but this was the most common syntactical error I helped with; either no comma or a semi-colon was used when I helped out.